schema.sql

```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE cheesecake;
+USE cheesecake;
+
+CREATE TABLE user(
```

Comment:
Kevin's (and many other people's opinion) that table names should be plural, because its a list. `users`

missing:

- [ ] email 
- [ ] password
- [ ] token varchar(255)